inc/API.php

```diff
@@ -30,7 +30,11 @@ public static function check_request_success($request, $function_args = [])
 
     if (is_wp_error($request)) {
       foreach ($request->errors as $notice => $message) {
-        peerboard_add_notice(sprintf('%s : %s', $notice, $message[0]), __FUNCTION__, 'error', $function_args);
+        if ($message[0] === 'cURL error 60: SSL certificate problem: certificate has expired') {
+          return 'ssl_fix';
```

Comment:
I would also add a notice here and instruction if possible

Reply:
Ready